libs/ui/src/hooks/use-toast.ts

```
@@ -0,0 +1,194 @@
+
+  return {
+    id: id,
+    dismiss,
+    update,
+  }
+}
+
+function useToast() {
+  const [state, setState] = React.useState<State>(memoryState)
+
+  React.useEffect(() => {
```

Biome (2.4.15)

[error] This hook specifies more dependencies than necessary: state.

(lint/correctness/useExhaustiveDependencies)